central/notifiers/awssh/notifier.go

```diff
@@ -167,15 +193,42 @@ func newNotifier(configuration configuration) (*notifier, error) {
 	return &notifier{
 		configuration: configuration,
 		securityHub:   securityhub.New(awss),
-		account:       config.GetAccountId(),
-		arn:           product.arnFormatter(config.GetRegion()),
+		account:       awssh.GetAccountId(),
+		arn:           product.arnFormatter(awssh.GetRegion()),
 		cache:         map[string]*storage.Alert{},
 		alertCh:       make(chan *storage.Alert),
 		initDoneSig:   concurrency.NewSignal(),
 		// stoppedSig intentionally omitted - zero value is "already triggered"
 	}, nil
 }
 
+func getCredentials(config configuration) (*storage.AWSSecurityHub_Credentials, error) {
+	if !env.EncNotifierCreds.BooleanSetting() {
+		return config.descriptor.GetAwsSecurityHub().GetCredentials(), nil
+	}
+
+	if config.descriptor.GetNotifierSecret() == "" {
+		return nil, errors.Errorf("encrypted notifier credentials for notifier '%s' empty", config.descriptor.GetName())
+	}
+	if config.cryptoCodec == nil {
```

Comment:
this and the next if check might be overkill

Reply:
Removed the empty check for key, but I think we can keep this one in-case someone called the NewEmail and similar funs with a nil value for codec.